packages/flutter_test/lib/src/_matchers_web.dart

```diff
@@ -31,17 +31,24 @@ const bool canCaptureImage = false;
 /// test is running in a web browser using conditional import.
 class MatchesGoldenFile extends AsyncMatcher {
   /// Creates an instance of [MatchesGoldenFile]. Called by [matchesGoldenFile].
-  const MatchesGoldenFile(this.key, this.version);
+  const MatchesGoldenFile(this.key, this.version, { this.isFlaky = false});
 
   /// Creates an instance of [MatchesGoldenFile]. Called by [matchesGoldenFile].
-  MatchesGoldenFile.forStringPath(String path, this.version) : key = Uri.parse(path);
+  MatchesGoldenFile.forStringPath(String path, this.version, { this.isFlaky = false}) : key = Uri.parse(path);
 
   /// The [key] to the golden image.
   final Uri key;
 
   /// The [version] of the golden image.
   final int? version;
 
+  /// Whether this matcher allows goldens that do not match the master golden to pass.
+  ///
+  /// If set to true, the underlying implementation of [goldenFileComparator]
+  /// should produce a screenshot and make it available for human review but not
```

Comment:
Nit: 
```suggestion
  /// will produce a screenshot and make it available for human review but not
```

Reply:
This class does not provide an implementation of `goldenFileComparato`, so it cannot guarantee this behavior. I meant this paragraph as a recommendation for implementors. It's up to the implementation to decide whether to respect this flag, what "human review" means (e.g. it may be something other that Skia Gold or local files).